app/controllers/two_factor_authentication/webauthn_verification_controller.rb

```diff
@@ -107,7 +107,8 @@ def analytics_properties
         context: context,
         multi_factor_auth_method: auth_method,
         webauthn_configuration_id: form&.webauthn_configuration&.id,
-        multi_factor_auth_method_created_at: form&.webauthn_configuration&.created_at,
+        multi_factor_auth_method_created_at: form&.webauthn_configuration&.
+          created_at&.strftime('%s%L'),
```

Comment:
Trying to sort where we landed in [the last thread](https://github.com/18F/identity-idp/pull/8922#discussion_r1286159362). I don't have a strong opinion whether we do a one-off vs. baked into the analytics class, but I would think we'd want the format to be consistent across all call-sites for this particular event. Would this change mean we're logging `multi_factor_auth_method_created_at` as a timestamp for WebAuthn, but as a date still for other MFA types? 

Could we make the format consistent for the event at least? I think this is what you had originally.

Reply:
Yeah, I think we landed on only changing `multi_factor_auth_method_created_at`?